odoo-shippable/scripts/build-image.sh

```diff
@@ -422,7 +422,7 @@ PG_NON_DURABILITY=1 PG_LOGS_ENABLE=1 python ${REPO_REQUIREMENTS}/linit_hook/trav
 
 # Install & Configure RVM
 curl -sSL https://rvm.io/mpapis.asc | gpg --import -
-\curl -sSL https://get.rvm.io | /bin/bash -s stable --ruby
+\curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | /bin/bash -s stable --ruby
```

Comment:
I prefer to use the official one but using the behaviour commented here:
https://github.com/rvm/rvm/issues/4068#issuecomment-309576824

Could you create it and add a TODO comment in order to fix it if the official is fixed?